src/planscape/boundary/management/commands/upload_boundary.py

```diff
@@ -12,44 +12,68 @@
 
 
 class Command(BaseCommand):
-    help = ('Uploads a boundary layer specified in the boundary configuration file '
-            'into the Planscape database.')
+    help = (
+        "Uploads a boundary layer specified in the boundary configuration file "
+        "into the Planscape database."
+    )
 
     def add_arguments(self, parser):
-        parser.add_argument('data_directory', nargs='?', type=str,
-                            help='Location of the boundary data.')
-        parser.add_argument('--boundary', nargs='?', type=str,
-                            help=('Name of the boundary to upload; if missing, all '
-                                  'boundaries are uploaded.'))
-        parser.add_argument('--configuration_file', nargs='?', type=str,
-                            help='Path to the boundary.json file; default is BASE_DIR/config/boundary.json.')
-        parser.add_argument('--force',
-                            type=bool, default=False, action=argparse.BooleanOptionalAction,
-                            help='Overwrite the boundary layer if it exists.')
-        parser.add_argument('--verbose',
-                            type=bool, default=True, action=argparse.BooleanOptionalAction,
-                            help='Show the commands to be run but do not run them.')
-        parser.add_argument('--strict',
-                            type=bool, default=True, action=argparse.BooleanOptionalAction,
-                            help='If false, ignore errors if a polygon cannot be converted and uploaded.')
+        parser.add_argument(
+            "data_directory", nargs="?", type=str, help="Location of the boundary data."
+        )
+        parser.add_argument(
+            "--boundary",
+            nargs="?",
+            type=str,
+            help=(
+                "Name of the boundary to upload; if missing, all "
+                "boundaries are uploaded."
+            ),
+        )
+        parser.add_argument(
+            "--configuration_file",
+            nargs="?",
+            type=str,
+            help="Path to the boundary.json file; default is BASE_DIR/config/boundary.json.",
+        )
+        parser.add_argument(
+            "--force",
+            type=bool,
+            default=False,
+            action=argparse.BooleanOptionalAction,
+            help="Overwrite the boundary layer if it exists.",
+        )
+        parser.add_argument(
+            "--verbose",
+            type=bool,
+            default=True,
+            action=argparse.BooleanOptionalAction,
+            help="Show the commands to be run but do not run them.",
+        )
+        parser.add_argument(
+            "--strict",
```

Comment:
Nit: this too seems oddly named.  Perhaps make this "--ignore_upload_errors" ?

Reply:
To be honest, this parameter was not being used. A failure on the upload would basically crash the command.

Reply:
Feel free to remove this then if it's not useful.

Reply:
Same thing as the `verbose`, to be honest. It is being used as the input of the LayerMapping utility. Basically if true, this will stop importing the data at first error.

Nicely couples with `transaction_mode="commit_on_success"`.

I will remove the argument, because we always want the tool to succeed or fail loudly.